ui/src/components/table/GroupingsTable.tsx

```diff
@@ -0,0 +1,129 @@
+'use client';
+import {
+    useReactTable,
+    flexRender,
+    getCoreRowModel,
+    getPaginationRowModel,
+    getFilteredRowModel,
+    getSortedRowModel,
+    SortingState
+} from '@tanstack/react-table';
+import { Table, TableBody, TableCell, TableHead, TableHeader, TableRow } from '@/components/ui/table';
+import ColumnSettings from '@/components/table/table-element/ColumnSettings';
+import GroupingsTableHeaders from '@/components/table/table-element/GroupingsTableHeaders';
+import PaginationBar from '@/components/table/table-element/Pagination';
+import GlobalFilter from '@/components/table/table-element/GlobalFilter';
+import SortArrow from '@/components/table/table-element/SortArrow';
+import { useState } from 'react';
+import { SquarePen } from 'lucide-react';
+import GroupingPathCell from '@/components/table/table-element/GroupingPathCell';
+import { GroupingPath } from '@/models/groupings-api-results';
+
+interface GroupingTableProps {
+    data: GroupingPath[];
+}
+
+const GroupingsTable = ({ data }: GroupingTableProps) => {
```

Comment:
Since `GroupingTableProps` is not used anywhere, you can just directly place the object into the parameter like this:
`const GroupingsTable = ({ data }: { data: GroupingPath[] }) => {`

Reply:
done